<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 35, 45); color: rgb(255,255,255);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Pre-processing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTreeWidget" name="overlay_FileTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QHeaderView::section {background-color: rgb(11, 76, 80);
color: rgb(215, 215, 215);
border-style: solid;
border-width:0px;
border-left-width: 1px;
padding: 2px;
padding-left: 5px;

}</string>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="itemsExpandable">
              <bool>true</bool>
             </property>
             <property name="headerHidden">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>3</number>
             </property>
             <attribute name="headerCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="headerMinimumSectionSize">
              <number>10</number>
             </attribute>
             <attribute name="headerDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Filename</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Cells</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Color</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="Smooth_Group">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="statusTip">
              <string>applying scipy savgol_filter</string>
             </property>
             <property name="accessibleDescription">
              <string/>
             </property>
             <property name="title">
              <string>KDE Settings</string>
             </property>
             <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0" columnstretch="0,0">
              <item row="4" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2"/>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="peak_error_label">
                <property name="text">
                 <string/>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QGroupBox" name="groupBox_5">
                <property name="title">
                 <string>positive Clustering</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="input_kMeansPositiv">
                   <property name="text">
                    <string>2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>populations</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="button_kMeansModelPositive">
                   <property name="text">
                    <string>Create Model</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="button_kMeansPredictPositive">
                   <property name="text">
                    <string>predict</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>GroupBox</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="input_Bandwidth">
                   <property name="inputMask">
                    <string/>
                   </property>
                   <property name="text">
                    <string>0.5</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QPushButton" name="button_refreshOverlay">
                   <property name="font">
                    <font>
                     <pointsize>8</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(11, 76, 80)</string>
                   </property>
                   <property name="text">
                    <string>Draw Overlay</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="comboBox_autoBandwidth">
                   <item>
                    <property name="text">
                     <string>none</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>scott</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>silverman</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label2">
                   <property name="text">
                    <string>auto Bandwidth</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Bandwidth</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QDialogButtonBox" name="dialog_buttons">
                   <property name="font">
                    <font>
                     <pointsize>8</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(11, 76, 80)</string>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="standardButtons">
                    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
                   </property>
                   <property name="centerButtons">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="1" rowspan="3">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Kmeans Clustering</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,2,0" columnstretch="0,0,1,0,0">
                 <item row="0" column="3" rowspan="3" colspan="2">
                  <widget class="MplWidgetNoToolbar" name="MplWidget_dataLengths" native="true">
                   <property name="styleSheet">
                    <string notr="true">margin: 5px;</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Crop Length</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>populations</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QPushButton" name="button_appendClustering">
                   <property name="text">
                    <string>append to cells</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="input_kMeans">
                   <property name="text">
                    <string>2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="button_kMeansModel">
                   <property name="text">
                    <string>Create Model</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QRadioButton" name="radioButton_selectedData">
                   <property name="text">
                    <string>use selected Data</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="input_CropLength">
                   <property name="text">
                    <string>900</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="button_kMeansPredict">
                   <property name="text">
                    <string>predict</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>metric</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QComboBox" name="comboBox_metric">
                   <item>
                    <property name="text">
                     <string>euclidean</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>dtw</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>softdtw</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Overlay</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="MplWidget" name="MplWidget_overlay" native="true">
            <property name="styleSheet">
             <string notr="true">margin: 5px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Density</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="MplWidget" name="MplWidget_overlay_density" native="true">
            <property name="styleSheet">
             <string notr="true">margin: 5px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplWidgetNoToolbar</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
